src/main/jte/publisher.jte

```diff
@@ -0,0 +1,12 @@
+@import edu.kit.hci.soli.dto.LayoutParams
+@import edu.kit.hci.soli.config.template.JteContext
+
+@param JteContext context
+@param LayoutParams layout
+
+@template.layout(page = context.page("layout.publisher"), context = context, layout = layout, content = @`
+    <div class="container">
+        <h1>${context.localize("layout.publisherInfo")}</h1>
+        <h2>TBD.</h2>
```

Comment:
A note about the origin of the project ("SOLI was developed as part of PSE by ...", see [here](https://github.com/kronos-et-al/MensaApp) for an example) and a link to the HCI impressum would be nice.